vicon2_driver/src/vicon2_driver.cpp

```diff
@@ -14,83 +14,72 @@
 //
 // Author: David Vargas Frutos <david.vargas@urjc.es>
 
-#include <string>
-#include <vector>
-#include <memory>
-
 #include "vicon2_driver/vicon2_driver.hpp"
-#include "lifecycle_msgs/msg/state.hpp"
-
-using std::min;
-using std::max;
-using std::string;
-using std::map;
-using std::stringstream;
 
 // Transform the Vicon SDK enumerations to strings
-string Enum2String(const ViconDataStreamSDK::CPP::Direction::Enum i_Direction)
+string Enum2String(const Direction::Enum i_Direction)
 {
   switch (i_Direction) {
-    case ViconDataStreamSDK::CPP::Direction::Forward:
+    case Direction::Forward:
```

Comment:
As I said before, I prefer to include all the namespace

Reply:
ok